lib/widgets/layouts/adaptive_layout/app_adaptive_scaffold_body.dart

```diff
@@ -6,18 +7,41 @@ import 'package:fluffychat/widgets/layouts/adaptive_layout/app_adaptive_scaffold
 import 'package:fluffychat/widgets/layouts/enum/adaptive_destinations_enum.dart';
 import 'package:fluffychat/widgets/matrix.dart';
 import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
 
 typedef OnOpenSearchPage = Function();
 typedef OnCloseSearchPage = Function();
 typedef OnClientSelectedSetting = Function(Object object, BuildContext context);
 typedef OnDestinationSelected = Function(int index);
 
-class AppAdaptiveScaffoldBody extends StatefulWidget {
+class AppAdaptiveScaffoldBodyArgs extends Equatable {
```

Comment:
why not abstract this argument?
```suggestion
abstract class AbsAppAdaptiveScaffoldBodyArgs extends Equatable {
```

Reply:
- `LoginBodyArgs`
- `SwitchActiveAccountBodyArgs`
- `LogoutBodyArgs`